### DISCUSSION TITLE
Possible to use the app with the balloon visible?

### DISCUSSION THREAD

**Discussion by Canato** [skydoves/Balloon#138] [Q&A] [answered]
When I show a ballon I cannot use the app (swipe, click, etc) until I dismiss the Balloon.

There is any settings that make possible to show the balloon but allow the rest of the UI keep responsive?

Thanks! 

@skydoves I tag you, cause there is no discussion before, so not sure if you would check it

**Comment by skydoves** [comment, marked as answer]
Hi, could you try the below code?

`setFocusable(false)`

**Reply by Canato** [reply]
Work like a charm, sorry for the newbie question ^^

And thanks for the fast answer

**Reply by asimkaka-coder** [reply]
@Canato @skydoves I am using setFocusable(false) with overlay set to visible and it is not working. Is there any way?
